tests/unit/manager/schemas.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { JobConfig } from '../../../src/manager/interfaces';
+import { validateJobConfig } from '../../../src/manager/schemas';
+
+describe('validateJobConfig', () => {
```

Comment:
add a test case for `uniqueItemProperties`

Reply:
added